Gateway/Http/TransferFactory.php

```diff
@@ -43,6 +43,7 @@ public function create(array $request)
         return $this->transferBuilder
             ->setBody($request)
             ->setMethod('POST')
+            ->setClientConfig(['timeout' => 60])
```

Comment:
This could be set like the GatewayUrl to allow the merchant to adjust the timeout in the admin settings instead off a hardcoded value.

Reply:
@mstein-ck Do you want to implement or leave it hard-coded at 60?

Reply:
I'd like to implement in admin settings but I don't think we're worse off with what we have now